scripts/integration-tests/prometheus/m3coordinator.yml

```diff
@@ -0,0 +1,45 @@
+listenAddress: 0.0.0.0:7201
```

Comment:
Is this never going to be useful for end users ? Maybe consider it moving out of integration-tests ?